pkg/controllers/managementuser/clusterauthtoken/token.go

```diff
@@ -33,10 +35,104 @@ type tokenHandler struct {
 	clusterUserAttribute       clusterv3.ClusterUserAttributeInterface
 	clusterUserAttributeLister clusterv3.ClusterUserAttributeLister
 	tokenIndexer               cache.Indexer
+	extTokenIndexer            cache.Indexer
 	userLister                 managementv3.UserLister
 	userAttributeLister        managementv3.UserAttributeLister
 }
 
+// ExtCreate is called when a given ext token is created, and is responsible for
+// updating/creating the ClusterAuthToken in a downstream cluster.
+func (h *tokenHandler) ExtCreate(token *extv1.Token) (*extv1.Token, error) {
+	_, err := h.clusterAuthTokenLister.Get(h.namespace, token.Name)
+	if !errors.IsNotFound(err) {
+		return h.ExtUpdated(token)
+	}
+
+	// we can sync tokens which are hashed by copying the hash downstream
+	// token is hashed, we can safely attempt to sync downstream
+	hashVersion, err := hashers.GetHashVersion(token.Status.Hash)
```

Comment:
Same as previous comment: ext tokens are always hased with sha3. Do we need this check?

Reply:
And the same answer, `extCreate` parallels `Create`. See line 155.